app/boards/intel_adsp_cavs25_tgph.conf

```diff
@@ -1,5 +1,6 @@
 CONFIG_TIGERLAKE=y
```

Comment:
typo: in git commit summary: s/thl-h/tgl-h/. 
Also as a style issue, the git summary should follow "FOO: BAR: blaa-baa" style, i.e. space after each prefix. We've not been very consistent with the "app/" folder commits so far, so this is not a big deal, but a note for future commits.